service/src/intTest/java/uk/nhs/adaptors/gp2gp/common/storageconnectors/AzureStorageConnectorTest.java

```diff
@@ -0,0 +1,65 @@
+package uk.nhs.adaptors.gp2gp.common.storageconnectors;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.List;
+import org.apache.commons.io.FileUtils;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class AzureStorageConnectorTest {
+    private static StorageConnector storageConnector;
+    private static final String FILE_TO_UPLOAD = "src/test/resources/test.txt";
+    private static final String FILE_EXPECTED_DOWNLOAD = "src/intTest/resources/downloads/test.txt";
+    private static final String SETUP_FILE = "setupFile.txt";
+    private static final String TEST_UPLOAD = "uploadFile.txt";
+
+    @BeforeAll
+    public static void setupConnector() throws IOException {
+        storageConnector = new AzureStorageConnector();
+        File file = new File(FILE_TO_UPLOAD);
+        InputStream inputStream = FileUtils.openInputStream(file);
+        storageConnector.uploadToStorage(inputStream, SETUP_FILE);
+    }
+
+    @Test
+    public void uploadToAzureStorage() throws IOException {
```

Comment:
Could we add some empty lines to make these tests more readable? Also, wha do you think about this test method naming?
"When_UploadingFileToAzureStorage_Expect_FileToBeAvaliableForDownload"

Reply:
I like that naming converntion, junit 5 also has a description for test we could us thoughts on that?

Reply:
The @mmackiewicz1 naming convention is also what we used on NHAIS